src/cursor/cur_hs.c

```diff
@@ -465,6 +509,11 @@ __curhs_next_visible(WT_SESSION_IMPL *session, WT_CURSOR_HS *hs_cursor)
         if (F_ISSET(std_cursor, WT_CURSTD_HS_READ_COMMITTED))
             break;
 
+        /*
```

Comment:
Can it be a one line comment?

Reply:
Converting this to a single line comment overflows it.